lib/src/_shared/icons/PenIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export const PenIcon: React.FC<SvgIconProps> = props => {
```

Comment:
You might want to look at the `createSvgIcon()` approach we have. It can be useful. For instance, to forward ref.

Reply:
I do think about using @material-ui/icons. I removed that dependency because there were a lot of problems with esm build. 
But AFAI everything should be ok right now?

Reply:
@oliviertassinari What is the approach you are talking about?

Reply:
Got it. It's about avoiding boilerplate. Looks good to me.
https://github.com/mui-org/material-ui/blob/89687f38cae750650555772ba4d821c9084d8dfc/packages/material-ui/src/internal/svg-icons/ArrowDropDown.js#L7

Reply:
I don't think that the picker should be importing from @material-ui/icons either. It's too big of a dependency, especially for people that would want to customize the picker. We don't use the icons package either in the @material-ui/core package.